### PR TITLE
Travis needs APCu for PHP 5.5+

### PR DESCRIPTION
**Pull request by dgrammatiko**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,7 +29,8 @@ before_script:
   - sh -c "if [ '$TRAVIS_PHP_VERSION' != '7.0' ]; then phpenv config-add build/travis/phpenv/memcached.ini; fi"
   - sh -c "if [ '$TRAVIS_PHP_VERSION' != '7.0' ]; then phpenv config-add build/travis/phpenv/apc-$TRAVIS_PHP_VERSION.ini; fi"
   - sh -c "if [ '$TRAVIS_PHP_VERSION' != '7.0' ]; then phpenv config-add build/travis/phpenv/redis.ini; fi"
-
+  # +5.5 uses APCU
+  - if [ "`phpenv version-name`" != "5.4" ] && [ "`phpenv version-name`" != "5.3" ] && [ "`phpenv version-name`" != "hhvm" ]; then printf "\n"| echo -e "extension = apcu.so\napc.enabled=1\napc.enable_cli=1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi
```
Comment (mbabker) [inline .travis.yml]: Can you use the TRAVIS_PHP_VERSION variable like the above checks and change the existing line to only load for 5.3 & 5.4?

Comment (dgt41) [inline .travis.yml]: 👍

Comment (dgt41) [inline .travis.yml]: Testing shows that travis with php5.5+ is faster without APCu. For the opcode these versions use OPcache so I don’t know if we really need it in the first place

Comment (mbabker) [inline .travis.yml]: We have code (cache and session) that supports APC so it should be tested
in all supported environments.  We don't have OPcache compatible stuff yet.

**Comment by dgt41** [comment]
11 commits for this is plain stupidity. Will do another one with APCu also for 5.5+

**Pull request closed without merging**